wear/src/main/java/org/gdg/frisbee/android/GdgWatchFace.java

```diff
@@ -462,6 +438,16 @@ private Paint getAdjustedPaintColor(Paint paint) {
             return paint;
         }
 
+        private String formatHour(int hour) {
+            int hourToDisplay = hour;
+            if(timeSetting == WearableConfigurationUtil.TIME_12_HOUR) {
+                hourToDisplay = (hour % WearableConfigurationUtil.TIME_12_HOUR) == 0 ?
+                    WearableConfigurationUtil.TIME_12_HOUR : hour%WearableConfigurationUtil.TIME_12_HOUR;
```

Comment:
There should be spaces around the 2nd `%`. I think we should enable checkstyle for this module too.